modules/channel-messenger/src/config.ts

```diff
@@ -0,0 +1,19 @@
+export interface Config {
+  /**
+   * The Facebook Page Access Token
+   */
+  verifyToken: string
+  /**
+   * The greeting message people will see on the welcome screen
+   */
+  greeting?: string
+  /**
+   * The message of the welcome screen "Get Started" button
+   */
+  getStarted?: string
```

Comment:
Maybe we could try to merge this information with the bot details page, so messenger/ channel web can use similar data

Reply:
I agree but get started and welcome message has not been added yet